app/controllers/posts_controller.rb

```diff
@@ -6,5 +6,21 @@ def index
   def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
+    @liked = Like.exists?(author: current_user, post: @post)
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(params.require(:post).permit(:title, :text))
```

Comment:
### **Optional**

- [ ] Please you can further secure your input by using the [Strong  Parameter](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md)

Reply:
@CollinsTatang, I thought I was using strong parameters by adding `params.require(:post).permit(:title, :text)`, do you mean moving this line to a private method?

Reply:
Yes @dicodiaz 
You can check out this [guide](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md) for more info.

Reply:
Gotcha, thank you very much